### PR TITLE
chore: Remove plugins docs and redirect to Hub

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

- Removes any links to plugins docs under https://www.cloudquery.io/ and redirects to https://hub.cloudquery.io/
- Fixes redirects from `/integrations` to the Hub. For example `https://www.cloudquery.io/integrations/jira/mysql` doesn't work (has 404). `https://www.cloudquery.io/integrations/jira` works. This PR fixes it so both work.

### CODE REVIEW

File: website/components/Destinations.tsx
Comment (erezrokah) [inline website/components/Destinations.tsx:1]: Not used

File: website/components/Plugins.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {ALL_PLUGINS, DESTINATION_CATEGORIES, SOURCE_CATEGORIES} from "./pluginData";
 
 import {LogoContainer} from "./LogoContainer";
-import {CATEGORIES, CategoryTitle} from "./Category";
```
Comment (erezrokah) [inline website/components/Plugins.tsx:4]: Not used

File: website/vercel.json [hunk 2]
```diff
@@ -747,116 +747,124 @@
       "destination": "https://hub.cloudquery.io/plugins/destination/cloudquery/sqlite"
     },
     {
-      "source": "/integrations/file",
```
Comment (erezrokah) [inline website/vercel.json:750]: Changes here sort the integrations redirects and fix redirects that have both source and destinations such as `/integrations/file/mysql`

File: website/scripts/prebuild.tsx
Comment (erezrokah) [inline website/scripts/prebuild.tsx:1]: Clean up some dead code

File: website/vercel.json [hunk 3]
```diff
@@ -747,116 +747,124 @@
...
-    {
-      "source": "/integrations/stripe",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/stripe"
+      "source": "/integrations/file/:path*",
+      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/file"
     },
     {
-      "source": "/integrations/terraform",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/terraform"
+      "source": "/integrations/gandi/:path*",
```
Comment (hermanschaaf) [inline website/vercel.json:766]: I don't think any other paths ever existed under the premium plugins, but it probably doesn't matter

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**